content/rs/administering/active-active.md

```diff
@@ -1,57 +1,57 @@
 ---
-Title: Active-Active Geo-Distributed Redis (formerly known as CRDB)
+Title: Active-Active Geo-Distributed Redis
 description:
 weight: $weight
 alwaysopen: false
 categories: ["RS"]
 aliases: /rs/administering/intercluster-replication/crdbs/
 ---
 In Redis Enterprise, active-active geo-distribution is based on [CRDT technology](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type).
-The Redis Enterprise implementation of CRDT is called Conflict-Free Replicated Database (CRDB) or Active-Active database.
-With CRDBs, applications can read and write to the same data set from different geographical locations seamlessly and with latency less than 1 ms,
+The Redis Enterprise implementation of CRDT is called an Active-Active database (formerly known as CRDB).
+With Active-Active databases, applications can read and write to the same data set from different geographical locations seamlessly and with latency less than 1 ms,
 without changing the way the application connects to the database.
 
-CRDBs also provide disaster recovery and accelerated data read-access for geographically distributed users.
+Active-Active databases also provide disaster recovery and accelerated data read-access for geographically distributed users.
 
 {{< note >}}
-CRDBs do not replicate the entire database, only the data.
+Active-Active databases do not replicate the entire database, only the data.
 Database configurations, Lua scripts, and other configurations are not replicated.
 {{< /note >}}
 
-You can create CRDBs on Redis Enterprise Software (RS) or Redis Cloud.
+You can create Active-Active databases on Redis Enterprise Software (RS) or Redis Cloud.
 
-## Considerations for Conflict-free Replicated Databases (CRDBs) {#considerations-for-conflictfree-replicated-databases-crdbs}
+## Considerations for Active-Active Databases {#considerations-for-activeactive-databases}
 
-CRDBs are based on multi-master replication that is configured to run on each database.
-A CRDB is made up of instances of the data that are each stored on an RS cluster.
+Active-Active databases are based on multi-master replication that is configured to run on each database.
+An Active-Active database is made up of instances of the data that are each stored on an RS cluster.
 
-Before configuring a CRDB, you must:
+Before configuring an Active-Active database, you must:
 
-- If the CRDB spans a WAN, establish a VPN between each networks that hosts a cluster with a CDRB instance.
-- Setup [RS clusters]({{< relref "/rs/administering/cluster-operations/new-cluster-setup.md" >}}) for each CRDB instance.
+- If the Active-Active database spans a WAN, establish a VPN between each networks that hosts a cluster with a CDRB instance.
```

Comment:
Are we still using the term CRDB instance?

Reply:
Missed it because it is misspelled :-)